README.md

```diff
@@ -16,7 +16,7 @@ Anime module for Nuxt.
 1. Install `@hypernym/nuxt-anime` to your project
 
 ```sh
-npm i -D @hypernym/nuxt-anime
+npx nuxi@latest module add animejs
 ```
 
```

Comment:
Thanks for the PR and suggestion.

I would leave the classic install as default, and maybe add this `nuxt` command right below?

```sh
npm i -D @hypernym/nuxt-anime
```
```sh
# via nuxi module
npx nuxi@latest module add animejs
```